backend/src/emcr/emcr.service.ts

```diff
@@ -126,7 +127,7 @@ export class EmcrService {
    * @returns {number} Count of total personnel search applies to
    */
   async getEmcrPersonnel(query: GetEmcrPersonnelDTO): Promise<{
-    personnel: EmcrPersonnelEntity[];
+    personnel: EmcrPersonnelEntity[] | PersonnelEntity[];
```

Comment:
You removed the equivalent from bcws service, correct?

Reply:
oh shoot one second I'll remove from here